### PR TITLE
Mathieu Juttet

### PR DESCRIPTION
**Pull request by MathieuJuttet**

### CODE REVIEW

**Comment by tomMoral** [comment]
If you are finished, you can sign up for this RAMP challenge: https://ramp.studio/events/brevet_datacamp2020
This is what we are going to do this afternoon. The goal is to try and solve the challenge.
You need to click on the blue button: `Join the event` and I need to approve your participation so send me a message once you have done this.
You can then read the starting kit and go to the left menu to submit your solutions in the sandbox.

File: pandas_questions.py [hunk 1]
```diff
@@ -38,8 +45,38 @@ def merge_referendum_and_areas(referendum, regions_and_departments):
...
+    referendum = referendum.loc[referendum['Department code'] != 'ZX']
+    referendum = referendum.loc[referendum['Department code'] != 'ZW']
+    referendum = referendum.loc[referendum['Department code'] != 'ZS']
+    referendum = referendum.loc[referendum['Department code'] != 'ZP']
+    referendum = referendum.loc[referendum['Department code'] != 'ZN']
+    referendum = referendum.loc[referendum['Department code'] != 'ZM']
+    referendum = referendum.loc[referendum['Department code'] != 'ZD']
+    referendum = referendum.loc[referendum['Department code'] != 'ZC']
+    referendum = referendum.loc[referendum['Department code'] != 'ZB']
+    referendum = referendum.loc[referendum['Department code'] != 'ZA']
```
Comment (tomMoral) [inline pandas_questions.py:58]: You can use `isin` to check if a string is not in a list and create a boolean mask.

**Review by tomMoral — CHANGES_REQUESTED** [review]
A small comment to make your code clearer but else it is quite nice!

File: pandas_questions.py (hunk 1 above)
Comment (MathieuJuttet) [reply pandas_questions.py]: an inner merge is solving this as I was trying to solve a pytest error

**Review by tomMoral — APPROVED** [review]
LGTM!

Indeed an inner merge is sufficient to avoid filtering. You could still have looked at how to do this with `isin` for the sake of it but good enough for me!

**Pull request closed without merging**